### ISSUE TITLE
[Python] Wheel for Raspberry PI?

### ISSUE THREAD

**Issue by YuukanOO** [snipsco/snips-nlu-parsers#15]
Hi there!

Just wondering if you have any plans to release pre-built wheels for the Raspberry PI?

I don't have any trouble to install it but compiling it takes a long time on a PI.

**Comment by pguyot** [comment]
@YuukanOO I agree this should be provided. It took 10 hours to build a wheel on a Pi Zero. Pis Zero have a smaller FPU instruction set than more powerful Pis, and judging from rust installation script this might make a difference, so I preferred to compile it directly on the target device.

I did upload the wheel here:
https://github.com/pguyot/snips-nlu-parsers/releases/tag/v0.3.0_SNAPSHOT

It should work on any Pi. I will also upload the wheel for snips-nlu-utils.

**Comment by leahcornelius** [comment]
how would one use the wheel?
